mailpoet/lib/EmailEditor/Integrations/MailPoet/PersonalizationTagManager.php

```diff
@@ -37,6 +42,19 @@ public function initialize() {
         __('Subscriber', 'mailpoet'),
         [$this->subscriber, 'getEmail'],
       ));
+      // Site Personalization Tags
+      $registry->register(new Personalization_Tag(
+        __('Site Title', 'mailpoet'),
+        'mailpoet/site-title',
+        __('Site', 'mailpoet'),
+        [$this->site, 'getTitle'],
+      ));
+      $registry->register(new Personalization_Tag(
+        __('Homepage URL', 'mailpoet'),
+        'mailpoet/site-homepage-url',
+        __('Site', 'mailpoet'),
+        [$this->site, 'getHomepageURL'],
+      ));
```

Comment:
Just an idea for the future... What about moving these two tags into the editor package? The logic behind this idea is similar to core blocks renderer being placed there. We could distribute the editor with a few basic or core tags.

Reply:
This sounds good to me. I think we can do it in a follow-up ticket where we can also add `date` tags.